graylog2-web-interface/src/components/sidecars/configurations/CopyConfigurationModal.jsx

```diff
@@ -43,47 +42,37 @@ class CopyConfigurationModal extends React.Component {
   };
 
   _save = () => {
-    const configuration = this.state;
+    const { copyConfiguration } = this.props;
+    const { errorMessage, id, name } = this.state;
 
-    if (!configuration.error) {
-      this.props.copyConfiguration(this.state.id, this.state.name, this._saved);
+    if (errorMessage) {
```

Comment:
```suggestion
    if (!errorMessage) {
```
Otherwise will do the opposite of what we want.